jetson/nav/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.8)
```

Comment:
can jarvis still build nav?

Reply:
Yes it builds on both. I can remove cmake if you'd like to keep things clean though

Reply:
i think it's fine to keep in, can you add a comment in the readme or smth explaining why both are there/when one would the cmake file